src/Pages/CategoriesPage.jsx

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
 
-const CategoriesPage = () => (
-  <div className="flex justify-center py-8">
-    <button type="button" className="border-2 border-fuchsia-500 p-2 cursor-pointer rounded-lg">Check Status</button>
-  </div>
-);
+const CategoriesPage = () => {
+  const dispatch = useDispatch();
+
+  return (
+    <div className="flex justify-center py-8">
+      <button onClick={() => dispatch(checkStatus)} type="button" className="border-2 border-fuchsia-500 p-2 cursor-pointer rounded-lg">Check Status</button>
+    </div>
```

Comment:
- [ ] Kindly render the text with `under construction ` after clicking on the check status button.

Reply:
Thankyou @omarsalem7 I have fixed this issue.

Reply:
- Well resolved 👍🏽